release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3402,6 +3402,35 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
+// 4.14.46
+[id="ocp-4-14-46_{context}"]
+=== RHSA-2025:0840 - {product-title} 4.14.46 bug fix and security update
+
+Issued: 06 February 2025
+
+{product-title} release 4.14.46, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0840[RHSA-2025:0840] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:0842[RHSA-2025:0842] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.46 --pullspecs
+----
+
+[id="ocp-4-14-46-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, crun failed to stop a container if you opened a terminal session and then disconnected from it. With this release, the issue is resolved.  (link:https://issues.redhat.com/browse/OCPBUGS-48752[*OCPBUGS-48752*])
+
+
+* Previously, you could not remove a finally pipeline task from the *edit Pipeline* form if you created a pipeline with only one finally task. With this release, you can remove the finally task from the *edit Pipeline* form and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-46603[*OCPBUGS-46603*])
```

Comment:
Just a question: should we format "finally" somehow, since it refers to a specific type of pipeline task?